### DISCUSSION TITLE
Is cache generating at `build` not redundant? How to off this feature?

### DISCUSSION THREAD

**Discussion by yurkimus** [parcel-bundler/parcel#5982] [Q&A] [answered]
I saw in docs that --no-cache flag is only for disable reading from cache.

I want to prevent cache generating on build, is it possible?

**Comment by DeMoorJasper** [comment, marked as answer]
That is no longer possible in Parcel 2 as we also use this to offload some data to reduce memory usage and ipc latency

**Reply by yurkimus** [reply]
Okay, thanks!
